examples/custom-reducer/.babelrc

```diff
@@ -28,6 +28,7 @@
     "@babel/plugin-proposal-nullish-coalescing-operator",
     "@babel/plugin-proposal-do-expressions",
     "@babel/plugin-proposal-function-bind",
-    "@babel/plugin-transform-modules-commonjs"
+    "@babel/plugin-transform-modules-commonjs",
+    ["inline-json-import", {}]
```

Comment:
need to bring this back to read package.json version value

Reply:
You don't need to  bring this back, cause the example as a standalone app should load from kepler.gl/dist, which doesn't not need to load package.json

Reply:
We are reading a json file from configuration folder and use it in __app.js__.

This example is broken in master branch